sources/packages/api/src/database/entities/sfas-part-time-application.model.ts

```diff
@@ -0,0 +1,68 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+import { BaseModel } from ".";
+import { TableNames } from "../constant";
+import { dateOnlyTransformer } from "../transformers/date-only.transformer";
+
+/**
+ * Data related to a Student Application on SFAS Part Time.
+ */
+@Entity({ name: TableNames.SFASPartTimeApplications })
+export class SFASPartTimeApplications extends BaseModel {
+  /**
+   * The unique key/number used in SFAS to identify this application (Sail_extract_data.sail_application_no).
+   */
+  @PrimaryColumn()
+  id: string;
+  /**
+   * The unique key/number used in SFAS to identify this individual (Sail_extract_data.individual_idx).
+   */
+  @Column({
+    name: "individual_id",
+    nullable: false,
+  })
+  individualId: number;
+  /**
+   * Educational program start date (Sail_extract_data.educ_start_dte).
+   */
+  @Column({
+    name: "start_date",
+    type: "date",
+    nullable: true,
+    transformer: dateOnlyTransformer,
+  })
+  startDate?: Date;
+  /**
+   * Educational Program End date (Sail_extract_data.educ_end_dte).
+   */
+  @Column({
+    name: "end_date",
+    type: "date",
+    nullable: true,
+    transformer: dateOnlyTransformer,
+  })
+  endDate?: Date;
+  /**
+   * CSGP Award Amount (sail_extract_data.sail_csgp_award_amt).
+   */
+  @Column({
```

Comment:
It seems that this is not in sync with the table. Can you please make the table and model in sync about nullable/not nullable?